x-pack/plugins/security_solution/public/management/cypress/e2e/serverless/feature_access/complete_with_endpoint.cy.ts

```diff
@@ -47,7 +47,10 @@ describe(
         });
       }
 
-      for (const actionName of RESPONSE_ACTION_API_COMMANDS_NAMES) {
+      // TODO: update tests when `scan` is included in PLIs
```

Comment:
Remind me: we have an issue tracking that, correct?

Reply:
Not a separate ticket but the same API ticket has an item in it.